clients/privacy-center/package-lock.json

```diff
@@ -1001,31 +1001,6 @@
         "@chakra-ui/theme-tools": "^1.3.1"
       }
     },
-    "node_modules/@chakra-ui/anatomy/node_modules/@chakra-ui/theme-tools": {
-      "version": "1.3.1",
-      "resolved": "https://registry.npmjs.org/@chakra-ui/theme-tools/-/theme-tools-1.3.1.tgz",
-      "integrity": "sha512-D8arJ5uFGuYZrrFGpXqgov8FhsJYWRyar5oBZY5TJR9gsVYBlJ8Ai91pwM/NflCFqzerTOgyt7bNSGQMdZ8ghA==",
-      "license": "MIT",
-      "dependencies": {
-        "@chakra-ui/utils": "1.9.1",
-        "@ctrl/tinycolor": "^3.4.0"
-      },
-      "peerDependencies": {
-        "@chakra-ui/system": ">=1.0.0"
-      }
-    },
-    "node_modules/@chakra-ui/anatomy/node_modules/@chakra-ui/utils": {
-      "version": "1.9.1",
-      "resolved": "https://registry.npmjs.org/@chakra-ui/utils/-/utils-1.9.1.tgz",
-      "integrity": "sha512-Tue8JfpzOqeHd8vSqAnX1l/Y3Gg456+BXFP/TH6mCIeqMAMbrvv25vDskds0wlXRjMYdmpqHxCEzkalFrscGHA==",
-      "license": "MIT",
-      "dependencies": {
-        "@types/lodash.mergewith": "4.6.6",
-        "css-box-model": "1.2.1",
-        "framesync": "5.3.0",
-        "lodash.mergewith": "4.6.2"
-      }
-    },
```

Comment:
Similar to previous comment, `@chakra-ui/utils` is included as a package further down (search for `"resolved": "https://registry.npmjs.org/@chakra-ui/utils/-/utils-1.10.0.tgz"`).